mteb/tasks/Clustering/pol/PolishClustering.py

```diff
@@ -21,22 +28,126 @@ class EightTagsClustering(AbsTaskClustering):
         eval_splits=["test"],
         eval_langs=["pol-Latn"],
         main_score="v_measure",
-        date=None,
-        form=None,
-        domains=None,
-        task_subtypes=None,
-        license=None,
-        socioeconomic_status=None,
-        annotations_creators=None,
-        dialect=None,
-        text_creation=None,
-        bibtex_citation=None,
-        n_samples=None,
-        avg_character_length=None,
+        date=("2019-01-01", "2020-05-01"),
+        form=["written"],
+        domains=["Social"],
+        task_subtypes=["Topic classification"],
```

Comment:
This is topical clustering, no?

Reply:
I'm copying that from the paper. We can add thematic clustering too.

Reply:
Done!